cl/sentinel/handlers/handlers.go

```diff
@@ -48,12 +75,13 @@ const (
 func NewConsensusHandlers(ctx context.Context, db persistence.RawBeaconBlockChain, host host.Host,
 	peers *peers.Pool, beaconConfig *clparams.BeaconChainConfig, genesisConfig *clparams.GenesisConfig, metadata *cltypes.Metadata) *ConsensusHandlers {
 	c := &ConsensusHandlers{
-		host:          host,
-		metadata:      metadata,
-		beaconDB:      db,
-		genesisConfig: genesisConfig,
-		beaconConfig:  beaconConfig,
-		ctx:           ctx,
+		host:           host,
+		metadata:       metadata,
+		beaconDB:       db,
+		genesisConfig:  genesisConfig,
+		beaconConfig:   beaconConfig,
+		ctx:            ctx,
+		peerRateLimits: make(map[string]map[string]*RateLimiter),
```

Comment:
this map can lead to crashes due to concurrency, use `sync.Map`; You shoukd also try to combine the two strings in one, an easy way would be to use `utils.Sha256` which should reduce it to a unique hash